hbase-common/src/main/java/org/apache/hadoop/hbase/io/encoding/RowIndexEncoderV1.java

```diff
@@ -47,7 +47,7 @@ public int write(Cell cell) throws IOException {
       rowsOffsetBAOS.writeInt(out.size() - startOffset);
     }
     lastCell = cell;
-    return encoder.write(cell);
+    encoder.write(cell);
```

Comment:
So u will handle the size update for this encoder in next push right?

Reply:
size update for this encoder is pending.